packages/app-staking/src/Overview/index.tsx

```diff
@@ -13,22 +12,29 @@ import { formatNumber } from '@polkadot/util';
 
 import CurrentList from './CurrentList';
 import Summary from './Summary';
+import { withApi, api } from '@polkadot/ui-api';
+import { ApiProps } from '@polkadot/ui-api/types';
 
-type Props = BareProps & ComponentProps & {
+type Props = ApiProps & BareProps & ComponentProps & {
   chain_subscribeNewHead?: HeaderExtended
 };
 
-const ZERO = new Balance(0);
-
 class Overview extends React.PureComponent<Props> {
   render () {
-    const { balances, chain_subscribeNewHead, controllers, recentlyOffline, validators } = this.props;
-    const nextSorted = this.sortByBalance(
-      controllers.filter((address) =>
-        !validators.includes(address)
-      )
-    );
-    const validatorsSorted = this.sortByBalance(validators);
+    const { chain_subscribeNewHead, allControllers, allStashes, recentlyOffline, currentValidatorsControllersV1OrStashesV2 } = this.props;
+    let nextSorted: Array<string>;
+
+    if (Object.keys(api.consts).length) {
```

Comment:
I'm wondering if we should add a boolean to `ui-api` for this - so we can just go `api.isSubstrate2` (or something like that - just worried we do this in a number of places)

Reply:
:+1: